### PR TITLE
Matches with changes in Maliput: Lane::ToLanePosition.

### PR DESCRIPTION
**Pull request by francocipollone**
Related to https://github.com/maliput/maliput/issues/496

### Summary

Matches with https://github.com/maliput/maliput/pull/521

action-ros-ci-repos-override: https://gist.githubusercontent.com/francocipollone/b218a7671b9bb6b6be2c22ac19f85793/raw/8c1e3852f99a5ea730b12be37ac6c6205cf67a2a/maliput_malidrive.repos

### CODE REVIEW

File: src/maliput_malidrive/base/lane.cc [hunk 1]
```diff
@@ -189,11 +189,23 @@ Vector3 Lane::BackendFrameToLaneFrame(const Vector3& xyz) const {
 
 void Lane::DoToLanePositionBackend(const maliput::math::Vector3& backend_pos, maliput::api::LanePosition* lane_position,
                                    maliput::math::Vector3* nearest_backend_pos, double* distance) const {
+  InertialToLaneSegmentPositionBackend(true, backend_pos, lane_position, nearest_backend_pos, distance);
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply src/maliput_malidrive/base/lane.cc]: Done.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Comment by francocipollone** [comment]
@ros-pull-request-builder retest this please

**Comment by francocipollone** [comment]
@ros-pull-request-builder retest this please

**Pull request merged**